### PR TITLE
Fixes for wrapperCol and labelCol, adding Bool field to demo app and docs update

### PR DESCRIPTION
**Pull request by janowsiany**
Fixing #207

### CODE REVIEW

**Comment by janowsiany** [comment]
I am not sure what else should be placed in docs. Any ideas?

File: API.md [hunk 1]
```diff
@@ -162,6 +162,20 @@ import BoolField from 'uniforms-unstyled/BoolField'; // Choose your theme packag
     grid={{md: 5}}  // 'col-5-md' on label, 'col-7-md' on input
     grid="col-6-xl" // 'col-6-xl' on label, 'col-6-xl' on input
 
+    // Field layout
+    //  The layout of label. You can set span and/or offset.
+    // Available in:
+    //   antd
+    labelCol={span: 4}   // 'ant-col-4' on label
```
Comment (radekmie) [inline API.md]: Here _(and all occurrences below)_: `{{span: 4}}` - it's an object.

File: packages/uniforms-antd/src/BoolField.js [hunk 2]
```diff
@@ -6,8 +6,8 @@ import filterDOMProps from 'uniforms/filterDOMProps';
 
 import wrapField from './wrapField';
 
-const Bool = props =>
-    wrapField(props, (
+const Bool = ({labelCol, wrapperCol, ...props}) =>
```
Comment (radekmie) [inline packages/uniforms-antd/src/BoolField.js]: In #229 we've added `colon` and `required` - these also should be treated like this. By the way, we should find a better way to handle these... How about `filterDOMProps.register` them?

**Comment by janowsiany** [comment]
`grouped` property is already added to API, you have even praised this [here](https://github.com/vazco/uniforms/pull/227/files#r104743614), haven't you?

File: packages/uniforms-antd/src/BoolField.js (hunk 2 above)
Comment (janowsiany) [inline packages/uniforms-antd/src/BoolField.js]: Right, i beleive `filterDOMProps.register` is cleaner solution here. Thanks that you point them to me.

**Comment by radekmie** [comment]
Well, yeah, my fault.

**Pull request merged**